common-build.gradle

```diff
@@ -47,3 +47,133 @@ task syncVersionAndroid  {
     }
 }
 
+task generateIOSBindings {
+    doLast() {
+        // Only generate bindings for iOS projects
+        if ( !project.hasProperty('iosSdkModuleName') ) {
+            return
+        }
+
+        println("Updating bindings for ${iosSdkModuleName} module.")
+
+        // clone iOS SDK at specified version, if it doesn't already exist
+        println("Using iOS SDK version ${airshipProperties.iosVersion}")
+        def tmpDir = System.getProperty("java.io.tmpdir")
+        def iosTempRepo = new File("${tmpDir}/ios-library-for-xamboni")
```

Comment:
will this be shared across builds?

Reply:
Yes. It is deleted at the end of the build, or when the daemon exits, whichever happens last.